src/pmod_als.erl

```diff
@@ -13,6 +13,10 @@
 %%% Vishay Semiconductor's TEMT6000X01.
 %%% http://www.vishay.com/docs/81579/temt6000.pdf
 %%%
+%%% Start the server with
```

Comment:
I think these should read:

> Start a device instance using
> ```
> 1> grisp:add_device(Slot, pmod_als).
> ```
> Where `Slot` is the actual slot the Pmod is connected to (note that some Pmods only work in some slots). For more information see [ref to `grisp` module, https://github.com/grisp/grisp/blob/master/src/grisp.erl#L19].

And we should probably list which ports each Pmod works with in the respective documentation, and have a proper list of slots in the `grisp` module docs.

Reply:
In this module the only working slot is `spi2`, so this is the listing of the possible slots for this module.

The slots are documented in the grisp module by "the lower case version of the slot names printed on the board itself". We can replace that by a list, but it works like this also.

Reply:
This doesn't start the device, does it? It starts the driver (i.e. the `gen_server`) communicating with the running device.

Reply:
I think the text should be the same for all drivers, and then we list the possible sorts (even when it's only one). That would be more consistent.

Reply:
Well, that it starts a "server" is an implementation detail (e.g. what would we write if it was `gen_statem` instead?). So let's call it driver then.

Reply:
> In this module the only working slot is `spi2`, so this is the listing of the possible slots for this module.
> 
> The slots are documented in the grisp module by "the lower case version of the slot names printed on the board itself". We can replace that by a list, but it works like this also.

Actually I’m assuming PMOD ALS work on both spi slots

Reply:
The one row SPI boards also work on the double row slots

Reply:
I think so too, but in the current code it's hard coded to `spi2`. So it is for most modules.
```
init([Slot = spi2, Opts]) ->
    ok = grisp_devices:register(Slot, ?MODULE),
    {ok, #state{options = Opts}};
init(Slot) ->
    error({incompatible_slot, Slot}).
```

Reply:
Oh, not aware of that.  Should probably fix that to spi1 and spi2 ... but no urgency.

Very sure it would work on spi1 too because its actually the same clock and data lines

Reply:
Yes, we can open an issue for that, I think. We should look through all Pmod driver and make the slots as flexible as possible. Here I will document the status quo.